e2e/pages/citizen/createCase/C100/otherProceedings/nonMolestationOrderOrderDetailsPage.ts

```diff
@@ -0,0 +1,233 @@
+import { NonMolestationOrderDetailsContent } from "../../../../../fixtures/citizen/createCase/C100/otherProceedings/nonMolestationOrderDetailsContent";
+import { Page } from "@playwright/test";
+import { Selectors } from "../../../../../common/selectors";
+import { Helpers } from "../../../../../common/helpers";
+import { CommonStaticText } from "../../../../../common/commonStaticText";
+import AccessibilityTestHelper from "../../../../../common/accessibilityTestHelper";
+
+interface NonMolestationOrderDetailsPageOptions {
+  page: Page;
+  accessibilityTest: boolean;
+  errorMessaging: boolean;
+  yesNoNonMolestationOrderDetails: boolean;
+}
+
+interface checkPageLoadsOptions {
+  page: Page;
+  accessibilityTest: boolean;
+}
+
+interface fillInFieldsOptions {
+  page: Page;
+  yesNoNonMolestationOrderDetails: boolean;
+}
+
+enum UniqueSelectors {
```

Comment:
do you want to define this enum in common file as its been reused 3 times?

Reply:
I can do